src/data/python-quiz.ts

```diff
@@ -1143,6 +1143,16 @@ const pythonQuiz = [
     Explanation: "+ is used to concatenate and * is used to multiply strings", 
     Link: "https://www.freecodecamp.org/news/learn-python-by-thinking-in-types", 
   },
+  {
+    Question: "Which sorting algorithm is used for built-in sort method of Python?",
+    Answer: "Tim Sort",
+    Distractor1: "Quick Sort",
+    Distractor2: "Merge Sort",
+    Distractor3: "Tim Sort",
+    Distractor4: "Insertion Sort",
+    Explanation: "Python's default sort uses Tim Sort, which is a combination of both merge sort and insertion sort.",
+    Link: "https://stackoverflow.com/questions/1517347/about-pythons-built-in-sort-method?noredirect=1&lq=1"
```

Comment:
Please use either freeCodeCamp resources, official documentation or as a last resort wikipedia. 
Avoid using stackoverflow answers because those can be deleted.